plugins/modules/ome_network_vlan_info.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell EMC OpenManage Ansible Modules
+# Version 2.1.3
+# Copyright (C) 2018-2020 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ome_network_vlan_info
+short_description: Retrieves the information about networks VLAN(s) present in OpenManage Enterprise.
+version_added: "2.10.0"
+description:
+    This module allows to retrieve the following
+    - A list of all the network VLANs with their detailed information.
+    - Information about a specific network VLAN using VLAN I(id) or VLAN I(name).
+extends_documentation_fragment:
+  - dellemc.openmanage.ome_auth_options
+options:
+    id:
+        description:
+            - A unique identifier of the network VLAN available in the device,
+            - I(id) and I(name) are mutually exclusive.
+        type: int
+    name:
+        description:
+            - A unique name of the network VLAN available in the device.
+            - I(name) and I(id) are mutually exclusive.
+        type: str
+
+requirements:
+    - "python >= 2.7.5"
+author: "Deepak Joshi(@deepakjoshishri)"
+'''
+
+EXAMPLES = """
+---
+- name: Retrieve information about all network VLANs(s) available in the device.
+  dellemc.openmanage.ome_network_vlan_info:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+
+- name: Retrieve information about a network VLAN using the VLAN ID.
+  dellemc.openmanage.ome_network_vlan_info:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    id: 12345
+
+- name: Retrieve information about a network VLAN using the VLAN name.
+  dellemc.openmanage.ome_network_vlan_info:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    name: "Network VLAN - 1"
+"""
+
+RETURN = '''
+---
+msg:
+  type: dict

```

Comment:
msg should be str

Reply:
Thanks for noting it out, Incorporated the comment.